contracts/contracts/ZodiacWands.sol

```diff
@@ -119,10 +119,12 @@ contract ZodiacWands is ERC721, GatedMint, Ownable {
 
   function unpack(uint256 tokenId) internal view returns (Wand memory) {
     Wand memory wand = Decanter.unpack(tokenId, wands[tokenId]);
-    wand.xp = address(forge) != address(0)
-      ? forge.xp(ERC721.ownerOf(tokenId))
-      : 0;
-    wand.level = address(forge) != address(0) ? forge.level(tokenId) : 0;
+
+    address owner = ownerOf(tokenId);
+    wand.level = forge.level(tokenId);
+    wand.xp.amount = forge.xp(owner) - forge.xpSpent(owner);
```

Comment:
This change assumes the Wand always has a forge configured. Because of the circular dependency it's not possible to deploy the wand with the Forge. In a real world prod setup yes the link will be there. However, shouldn't we in any case account for it and have a fallback? Example, some tests cases were forge agnostic, and I wasn't setting up a forge then